src/pages/SwapNew/components/CurrencyItem/CurrencyAmountWorth.tsx

```diff
@@ -0,0 +1,27 @@
+import { CurrencyAmount, Percent } from '@swapr/sdk'
+
+import styled from 'styled-components'
+
+import { TEXT_COLOR_SECONDARY } from '../../constants'
+import { FontFamily } from '../../constants'
+
+type CurrencyAmountWorthProps = {
+  fiatValue?: CurrencyAmount | null
+  priceImpact?: Percent
+  isFallback?: boolean
+}
+
+export function CurrencyAmountWorth({ fiatValue, priceImpact, isFallback }: CurrencyAmountWorthProps) {
+  const value = fiatValue ? `${`${isFallback ? '~' : ''}`}$${fiatValue.toFixed(2, { groupSeparator: ',' })}` : ''
+
+  return <Paragraph>{value}</Paragraph>
+}
+
+const Paragraph = styled.p`
+  line-height: 12px;
+  font-size: 10px;
+  ${FontFamily}
```

Comment:
Redundant `FontFamily` here too.

Reply:
Yeah, I guess if it's the same across the whole application, it is redundant on all of the places. 👍🏼